tablo/geom_utils.py

```diff
@@ -437,4 +437,196 @@ def extract_significant_digits(number):
     if is_negative:
         rounded = 0 - rounded
 
-    return rounded
\ No newline at end of file
+    return rounded
+
+
+def wkt_to_esri_feature(wkt):
+    geom_type = wkt[:wkt.find('(')]
+
+    def _geom_repl():
+        bracket_multiplier = 1
+        if geom_type.find('LINESTRING') > -1:
+            bracket_multiplier = 2
+        return json.loads(re.sub(
+            WKT_GEOM_2D_REGEX,
+            lambda m: '[{}]'.format(m.group().replace(' ', ',')),
+            wkt.replace(geom_type, '')
+        ).replace('(', '[' * bracket_multiplier).replace(')', ']' * bracket_multiplier))
+
+    if geom_type == 'POINT' or geom_type == 'MULTIPOINT':
+        match = WKT_POINT_REGEX.match(wkt)
+        if not match:
+            raise ValueError('Invalid Point Geometry: {0}'.format(wkt))
+        x, y = (float(x) for x in match.groups())
+        return {'x': x, 'y': y}
+    elif geom_type == 'LINESTRING' or geom_type == 'MULTILINESTRING':
+        return {'paths': _geom_repl(wkt, geom_type)}
+    elif geom_type == 'POLYGON' or geom_type == 'MULTIPOLYGON':
+        return {'rings': _geom_repl(wkt, geom_type)}
+
+
+def esri_feature_to_ewkt(feature, geom_type):
+    srid = 'srid={};'.format(feature.get('spatialReference', {}).get('wkid', 3857))
+    if geom_type == 'esriGeometryPoint':
+        return '{srid}POINT({x} {y})'.format(srid=srid, x=feature['x'], y=feature['y'])
+    elif geom_type == 'esriGeometryPolyline':
+        paths_text = json.dumps(feature['paths'])
+        lines = re.sub(
+            ESRI_GEOM_2D_REGEX,
+            lambda m: m.group().replace('[', '').replace(']', '').replace(',', ' '),
+            paths_text
+        ).replace('[', '(').replace(']', ')')
+        return '{srid}MULTILINESTRING{lines}'.format(srid=srid, lines=lines)
+    elif geom_type == 'esriGeometryPolygon':
+        rings_text = json.dumps(Terraformer.rings_to_wkt(feature['rings'])['coordinates'])
+        polygons = re.sub(
+            ESRI_GEOM_2D_REGEX,
+            lambda m: m.group().replace('[', '').replace(']', '').replace(',', ' '),
+            rings_text
+        ).replace('[', '(').replace(']', ')')
+        return '{srid}MULTIPOLYGON{polygons}'.format(srid=srid, polygons=polygons)
+    raise ValueError('Unsupported geometry type')
+
+
+class Terraformer:
```

Comment:
It looks like you are mostly using this as a namespace rather than a class to be instantiated.  How about moving this into a separate python file `wkt.py` which then gives you clarity and namespace in one, and allows you to drop all the static methods...